engine/src/shell/platform/unity/unity_console.h

```diff
@@ -0,0 +1,27 @@
+#include "runtime/mono_api.h"
```

Comment:
we should add a unity_console.cc too.

declare the methods in .h and put all the implementations in .cc